builders/tools/precompiles/xc20.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Cross-Chain Assets
+description:  Learn how to access and interact with an ERC-20 interface for cross-chain tokens on Moonbeam, and check how this is done on the Moonbase Alpha TestNet by using the assets precompiled contract.
+---
+
+#  Cross-Chain Assets Precompile
+
+![Cross-Chain Assets Precompiled Contracts Banner](/images/builders/tools/precompiles/xc20/xc20-banner.png)
+
+## Introduction {: #introduction } 
+
+The [Cross-Consensus Message (XCM)](https://wiki.polkadot.network/docs/learn-crosschain) format defines how messages can be sent between blockchains. This format opens the door to transfer messages and assets between Moonbeam and the Relay Chain or other parachains.
+
+Cross-chain assets on Moonbeam are one-to-one representations of the origin chain’s native tokens and are known as XC-20s. For example, Kusama's KSM representation will be known as XC-KSM. XC-20s allow users to bring their tokens to Moonbeam and interact with them through a standard ERC-20 interface via the [assets precompile](https://github.com/PureStake/moonbeam/blob/master/precompiles/assets-erc20/ERC20.sol).
```

Comment:
The first sentence isn't 100% true, they "should" be instead of "are" one-to-one representations